compiler_opt/rl/regalloc/gin_configs/ppo_nn_agent.gin

```diff
@@ -19,6 +19,7 @@ train_eval.train_sequence_length=16
 train_eval.deploy_policy_name='saved_collect_policy'
 train_eval.moving_average_decay_rate=0.8
 train_eval.use_random_network_distillation=False
+train_eval.num_modules=512
```

Comment:
also add the flag to inlining ppo gin file

Reply:
With the same value?